streaming/README.md

```diff
@@ -1,5 +1,150 @@
 # Ray Streaming
```

Comment:
Let's create as `.rst` so that it matches the rest of our documentation?

Reply:
done, please have a look again.